src/test/scala/org/broadinstitute/dsde/workbench/sam/service/TestPolicyEvaluatorServiceBuilder.scala

```diff
@@ -0,0 +1,62 @@
+package org.broadinstitute.dsde.workbench.sam.service
+
+import cats.effect.IO
+import org.broadinstitute.dsde.workbench.openTelemetry.OpenTelemetryMetrics
+import org.broadinstitute.dsde.workbench.sam.api.TestSamRoutes.SamResourceActionPatterns
+import org.broadinstitute.dsde.workbench.sam.dataAccess.{AccessPolicyDAO, DirectoryDAO, StatefulMockAccessPolicyDaoBuilder}
+import org.broadinstitute.dsde.workbench.sam.model._
+import org.mockito.scalatest.MockitoSugar
+
+import scala.collection.mutable
+import scala.concurrent.ExecutionContext
+
+case class TestPolicyEvaluatorServiceBuilder(
+    directoryDAO: DirectoryDAO,
+    policyDAOOpt: Option[AccessPolicyDAO] = None,
```

Comment:
```suggestion
    maybePolicyDao: Option[AccessPolicyDAO] = None,
```
there is a bit of a convention for `Option`s to be called `maybe…`

Reply:
Removed the `Option` type, not needed anyway.